packages/modules/link-modules/src/definitions/readonly/cart-product.ts

```diff
@@ -0,0 +1,43 @@
+import { Modules } from "@medusajs/modules-sdk"
+import { ModuleJoinerConfig } from "@medusajs/types"
+
+export const CartProduct: ModuleJoinerConfig = {
```

Comment:
Got it, that's interesting. It was not immediately obvious to me, that this was extending items.variant. How is that possible without any mention of items in the joiner config?

Reply:
at the moment, without the GraphQL definition a link is module to module, so it will try to get the defined property alias in any "level".
this is why we have to also add the "args.methodSuffix" to it.
When we have the GQL definition, we gonna link at the Entity level directly. then it would be from the entity CartItem to ProductVariant.

Reply:
btw `foreignKey: "items.variant_id"` also works
I will keep it like this so it is more explicity while we don't have the graph of all modules.

Reply:
This is easier to read imo 👍